test/protocol-buffer-conversion-test.ts

```diff
@@ -25,6 +30,7 @@ testPathElement.setAccount(testAccountAddress)
 testPathElement.setIssuer(testAccountAddressIssuer)
 
 describe('Protocol Buffer Conversion', function(): void {
+  // Currency
```

Comment:
Nit: I'd put on vertical space above and below these headers so they're more visible.

Feel free to ignore if you've seen this style elsewhere in Xpring's JS Codebase